mw/diag/dtc/service/dtcService.hpp

```diff
@@ -23,50 +26,54 @@
 #include "communication-core/sockets/ipc_socket.h"
 #include "communication-core/sockets/socket_config.h"
 #include "core/application/application_mw.h"
-#include "core/application/application_factory.h"  // <- to psuło ale już jest naprawione
+#include "core/application/application_factory.h"
 #include "core/logger/ILogger.h"
 #include "diag/dtc/factories/dtc_msg_factory.hpp"
 #include "diag/base/data/data_structure.h"
 #include "diag/base/data/parser.h"
 #include "diag/base/factories/diag_data_factory.h"
 #include "diag/base/controller/diag_controller.h"
 
+#define MAX_STRING_LENGTH 7
+
```

Comment:
a nie lepiej to co ci ostatnio pokazywałem ? to trochę słabo wygląda.

Reply:
ale może zostać oczywiście

Reply:
mogło by być, 2 razy użyłem tego, ale nie pomyślałem tutaj o tym

Reply:
pozatym myślałem że nie przepuścisz jeszcze